tests/synchronous_mode_test.py

```diff
@@ -8,10 +12,7 @@ def setup_class(cls):
 
     def __check_result(self, result_msg, mode):
         if mode == "enable" or mode == "disable":
-            expected_msg = """Wrote %s synchronous mode into CONFIG_DB, swss restart required to apply the configuration: \n
-    Option 1. config save -y \n
-              config reload -y \n
-    Option 2. systemctl restart swss""" % mode
+            expected_msg = "Wrote %s synchronous mode into CONFIG_DB, swss restart required to apply the configuration" % mode
```

Comment:
>expected_msg [](http://example.com/codeflow?start=12&length=12)

What is the reason to shorten the `expected_msg`?

Reply:
I initially had some white space inconsistencies that I have since fixed, reverted back to the original `expected_msg`